src/components/sortableList/index.tsx

```diff
@@ -84,7 +84,8 @@ const SortableList = <ItemT extends SortableListItemProps>(props: SortableListPr
       enableHaptic,
       scale
     };
-  }, [data]);
+  }, [data, onChange]);
```

Comment:
Did this cause a bug (it looks like this is only affected by the user changing `onOrderChange`)?

Reply:
You mean it's on purpose? 
But if the user changes the onOrderChange, it's not implausible